db/migrate/20191120000001_add_default_organizations.rb

```diff
@@ -0,0 +1,16 @@
+class AddDefaultOrganizations < ActiveRecord::Migration[5.1]
+    def up
+
+      # return if it's a new setup
+      return if !Setting.find_by(name: 'system_init_done')
+
+      Organization.create_if_not_exists(
```

Comment:
I believe that created_by_id and updated_by_id are required fields and update will not work if you don't specify them in migration.
Have you tested update scenario?